bubblescan-client/src/components/FileUploadComponent.tsx

```diff
@@ -24,7 +30,7 @@ function FileUploadComponent() {
   const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
```

Comment:
I would recommend adding more comments throughout the file explaining what the code is doing because that would make it easier to understand since I'm unfamiliar with typescript.

Reply:
Yes you can add more comments, just a brief about new functionality. But this is a small change so it can be exception. But it's encouraged to do so, this will help you be a good developer and others to understand as well. Still Good Work.